docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md

```diff
@@ -149,6 +149,8 @@ console.log(outputString);
 
 Gets an object that provides methods to get or update the recipients on the Bcc (blind carbon copy) line of a message. Compose mode only.
 
+By default, the collection is limited to a maximum of 100 members. However, on Windows and Mac, you can set up to 100 members per call to 500 total, and get 500 members maximum.
```

Comment:
The "However, " sentence is puzzling. is it supposed to mean "**up** to 500 total"? Also,, why is the 500 maximum being specified twice?